From ab4dae76909707192e4ec594368669a2918f0608 Mon Sep 17 00:00:00 2001
From: Jianqun xu <jay.xu@rock-chips.com>
Date: Tue, 1 Dec 2015 20:23:34 +0800
Subject: [PATCH] clk: rockchip: rk3368: add dt bindings for DPHY clocks

Add two clock IDs for DPHY clocks of Rockchip rk3368 SoC.

Change-Id: Iaae4da4a0ea9f4dee2b04fb8eb4f9400bd86511f
Signed-off-by: Jianqun xu <jay.xu@rock-chips.com>
---
 include/dt-bindings/clock/rk3368-cru.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/dt-bindings/clock/rk3368-cru.h b/include/dt-bindings/clock/rk3368-cru.h
index 9c5dd9ba2f6c..fc22f02bc3c4 100644
--- a/include/dt-bindings/clock/rk3368-cru.h
+++ b/include/dt-bindings/clock/rk3368-cru.h
@@ -151,6 +151,8 @@
 #define PCLK_ISP		366
 #define PCLK_VIP		367
 #define PCLK_WDT		368
+#define PCLK_DPHYRX		369
+#define PCLK_DPHYTX0		370
 
 /* hclk gates */
 #define HCLK_SFC		448
-- 
2.35.3

